cmd/atlas/internal/sqlparse/parseutil/parseutil.go

```diff
@@ -64,8 +64,8 @@ func RenameIndex(modify *schema.ModifyTable, r *Rename) {
 
 // RenameTable patches DROP/ADD table commands to RENAME.
 func RenameTable(changes schema.Changes, r *Rename) schema.Changes {
-	i := changes.IndexDropTable(r.From)
-	j := changes.IndexAddTable(r.To)
+	i := changes.LastIndexDropTable(r.From)
+	j := changes.LastIndexAddTable(r.To)
```

Comment:
```suggestion
	var (
		i = changes.LastIndexDropTable(r.From)
		j = changes.LastIndexAddTable(r.To)
	)
```

Reply:
I actually don't like to use var block here, because I use them in the`if` right after.